README.win32.adoc

```diff
@@ -29,7 +29,8 @@ Here is a summary of the main differences between these ports:
 (*):: Cygwin-generated `.exe` files are linked with the Cygwin DLL. Since
 version 2.5.2 released in June 2016, this DLL is licensed under the LGPLv3 with
 a static linking exception; prior versions of the DLL are distributed under the
-GPL. Thus, these `.exe` files can only be distributed under compatible license.
+GPL (the cygwin version can be obtained with the `uname -r` command). Thus,
+these `.exe` files can only be distributed under a compatible license.
```

Comment:
Alternatively, we could avoid being too specific (IANAL and all that), and simply state that distributing the produced `.exe` must be done in a way that is compliant with the Cygwin DLL license and leave it at that. That was my intent in leaving the sentence as is, but @gasche is right, the wording doesn't quite say that.